### PR TITLE
fix: ensure that composer is installed with minimum php version 5.6

### LINKED ISSUE #70: fix: ensure that composer is installed with minimum php version 5.6
# Bug Report

## User Story
As a developer, I want to install a composer when the PHP version is 5.6 so that it will support PHP versions 5.6 and more.

## Current Behavior
I'm getting error while running `composer install` as shown in the screenshot
image

## Expected Behavior
I expect to run the composer successfully when the PHP version is 5.6 or more.

## Bug Type
- [ ] This bug describes functionality that once worked as expected in version X.X.X.
- [ ] This bug describes functionality that never worked as expected.
- [x] I am not sure whether this functionality ever worked as expected.

## Steps to Reproduce
1. Set PHP 5.6
2. Clone GiveWP Campaigns repository
3. Run `composer install`
4. Notice the error.

## Acceptance Criteria
- [ ] Composer should run properly with PHP 5.6 and above versions

### PR DESCRIPTION
**Pull request by ravinderk**
## Description
This PR will resolve #70.

## How Has This Been Tested?
Tested locally by running `composer install`

## Types of changes

Bug fix (non-breaking change which fixes an issue)

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

**Comment by JasonTheAdams** [comment]
I'm really intrigued by this. I've never seen this before. I've done some research and understand the concept, but I'm not clear as to how it actually plays out. Do you have a specific problem or use case in mind for this, @ravinderk?

**Comment by ravinderk** [comment]
> I'm really intrigued by this. I've never seen this before. I've done some research and understand the concept, but I'm not clear as to how it actually plays out. Do you have a specific problem or use a case in mind for this, @ravinderk?

@JasonTheAdams Yes, I have a user case. I set up the composer with PHP 7.2, so composer installed packages which compatible was with that PHP 7.*  but since we support PHP 5.6, it must work on a lower version too.

So if you run `composer install` in PHP 5.6 or PHP < 7 then the process will stop in middle.

File: composer.json [hunk 1]
```diff
@@ -1,8 +1,13 @@
 {
+    "config": {
+        "platform": {
+            "php": "5.6"
+        }
+    },
```
Comment (knowler) [inline composer.json]: How is this different than the following?

```json
{
  "dependencies": {
    "php": ">=5.6"
  }
}
```

Comment (JasonTheAdams) [reply composer.json]: I wondered the same thing. Adding PHP as a dependency adds a check when comparing to other packages. But going the `platform` route actually "tricks" Composer (and other packages/frameworks that support this) into treating the local CLI as if it's using the specified PHP version. They overlap but are different.

**Pull request merged**